README.md

```diff
@@ -1,4 +1,4 @@
-# Project Mongo API
+# Project Mongo API (incomplete)
 
 Replace this readme with your own information about your project.
```

Comment:
Write a little something about the task, and then also about what endpoints the user can expect :)